<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjWindow</class>
 <widget class="QMainWindow" name="ObjWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_read">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>341</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>打开</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>300</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>返回主页</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>151</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>310</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionA"/>
    <addaction name="actionB"/>
    <addaction name="actionC"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>图像质量评价</string>
    </property>
    <addaction name="actionD"/>
    <addaction name="actionE"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>算法实验</string>
    </property>
    <addaction name="actionF"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionA">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionB">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionC">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>Full-reference</string>
   </property>
  </action>
  <action name="actionE">
   <property name="text">
    <string>No-reference</string>
   </property>
  </action>
  <action name="actionF">
   <property name="text">
    <string>进行实验</string>
   </property>
  </action>
  <action name="actionG">
   <property name="text">
    <string>PSNR</string>
   </property>
  </action>
  <action name="actionH">
   <property name="text">
    <string>SSIM</string>
   </property>
  </action>
  <action name="actionI">
   <property name="text">
    <string>MAD</string>
   </property>
  </action>
  <action name="actionJ">
   <property name="text">
    <string>NIQE</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
